### DISCUSSION TITLE
Can't add atorch (AT2PL/GR2P)

### DISCUSSION THREAD

**Discussion by BKSnake** [make-all/tuya-local#2071] [General]
I have AT2PL and when I tried to add it via `Configure your Tuya Local device` and see it as simple device and don't see full ability to configuration

<img width="333" alt="Screenshot 2024-07-02 at 22 12 43" src="https://github.com/make-all/tuya-local/assets/15376959/5c808cf9-39fd-470e-ab0d-5de121cae091">
Home assistan run under docker

**Comment by BKSnake** [comment]
{
  "type": "form",
  "flow_id": "cfad5df6a63dd021a4de2fc31f326c9b",
  "handler": "tuya_local",
  "data_schema": [
    {
      "type": "select",
      "options": [
        [
          "feit_light_rgb_bulb",
          "feit_light_rgb_bulb"
        ],
        [
          "hoenyzy_valve_timer",
          "hoenyzy_valve_timer"
        ],
        [
          "inkbird_ibs_m1s",
          "inkbird_ibs_m1s"
        ],
        [
          "ir_remote_sensors",
          "ir_remote_sensors"
        ],
        [
          "peteme_smart_recessed_lighting",
          "peteme_smart_recessed_lighting"
        ],
        [
          "pj1103_clamp_meter",
          "pj1103_clamp_meter"
        ],
        [
          "simple_contact_sensor",
          "simple_contact_sensor"
        ],
        [
          "simple_switch",
          "simple_switch"
        ],
        [
          "simple_switch_timerv2",
          "simple_switch_timerv2"
        ]
      ],
      "name": "type",
      "required": true,
      "default": "pj1103_clamp_meter"
    }
  ],
  "errors": null,
  "description_placeholders": null,
  "last_step": null,
  "preview": null,
  "step_id": "select_type"
}


why i didn't see `atorch_at2pl_breaker`

**Reply by tiasss** [reply]
did you manage to solve it? Can you share the information please?

**Comment by MarkiyanSHCH** [comment]
Did you fixed that?

**Comment by Nevynnyi** [comment]
my atorch_gr2pws_breaker.yaml

may be helps you. just rename atorch_gr2pws_breaker.txt to atorch_gr2pws_breaker.yaml and put it to /homeassistant/custom_components/tuya_local/devices directory

[atorch_gr2pws_breaker.txt](https://github.com/user-attachments/files/16985654/atorch_gr2pws_breaker.txt)

Еще при выборе устройства, когда добавляешь - выбери из списка свой нужный девайс. Если сразу не видет - попробуй несколько раз.


![Opera Снимок_2024-09-13_003945_local myleben net](https://github.com/user-attachments/assets/7c9e7470-ce05-40c9-af46-f310c41f0cb6)

**Comment by Luc4Piccinini** [comment]
I have the same problem. I have 2 Atorch GR2PWS. On Tuya app they add and configure fine and no problem. When you go to local tuya though it doesn't see them. If you add them manually putting all parameters including local key etc. When you finish the configuration the sensors all show as “unavailable”. 
Whoever suggested to put the file in devices inside local_tuya needs to explain better what to do. 
Also I understand that you have to have an esphome or you don't need it? 
I tried adding the posted file as mentioned but I don't see any change or appearance of the device with working sensors.
![2024-12-04 20_18_05-Impostazioni – Home Assistant](https://github.com/user-attachments/assets/e2bea799-4200-444f-b72c-dba03c6ba5ce)

**Reply by Nevynnyi** [reply]
Try tuya local in hacs, not local tuya
